ldcbench.api/src/main/java/org/dice_research/ldcbench/rdf/TripleCreator.java

```diff
@@ -12,4 +12,6 @@
 public interface TripleCreator {
 
     public Triple createTriple(int sourceId, int propertyId, int targetId, int targetExtId, int targetExtGraphId);
+
+    public Triple createTripleWithBlankNode(int sourceId, int propertyId, int targetId, int targetExtId, int targetExtGraphId);
```

Comment:
Why this additional method? In the other creator classes, you simply added a boolean flag to the create method. Here, you propose a completely new method although it mainly does what the `createTriple` method already does :thinking:

Reply:
the new method is there to differentiate the two cases and know when to set the blankNode param to true when calling createNode function. Another way of doing this could be adding an extra boolean param to the createTriple Method. But it would also change the way this method is used in the rest of the code.

Reply:
Please apply the same idea as in this comment: https://github.com/dice-group/orca/pull/82#discussion_r744787842